dashboard/app/models/user.rb

```diff
@@ -975,12 +976,12 @@ def teacher_email_required?
   end
 
   def email_or_hashed_email_required?
+    return false if Policies::Lti.lti? self
```

Comment:
Since teacher's emails are optional on the LMS side, during a user's creation it is now possible for the email to be `nil` but we will prompt the teacher for their email upon login.